tutorial/CMakeLists.txt

```diff
@@ -131,6 +131,7 @@ if(TUTORIALS_CHAPTER6)
     add_subdirectory("606_AmbientOcclusion")
   endif()
   if(LIBIGL_WITH_PNG)
+    add_subdirectory("111_MatCap")
```

Comment:
Ok maybe it's time for https://github.com/libigl/libigl.github.io/issues/44 as well? xD